src/debugger.hpp

```diff
@@ -474,8 +474,6 @@ inline void debug_ast(AST_Node_Ptr node, std::string ind, Env* env)
     std::cerr << " (" << pstate_source_position(node) << ")";
     std::cerr << " [indent: " << ruleset->tabs() << "]";
     std::cerr << (ruleset->is_invisible() ? " [INVISIBLE]" : "");
-    std::cerr << (ruleset->at_root() ? " [@ROOT]" : "");
-    std::cerr << (ruleset->is_root() ? " [root]" : "");
```

Comment:
nitpick: should this be kept since there is still the `is_root` flag still exists?